HISTORY.md

```diff
@@ -19,6 +19,8 @@ Our backwards-compatibility policy can be found [here](https://github.com/python
   and {func}`cattrs.gen.typeddicts.make_dict_structure_fn` will use the value for the `use_alias` parameter from the given converter by default now.
   If you're using these functions directly, the old behavior can be restored by passing in the desired value directly.
   ([#596](https://github.com/python-attrs/cattrs/issues/596) [#660](https://github.com/python-attrs/cattrs/pull/660))
+- Fix unstructuring of generic classes with stringified annotations 
```

Comment:
> Please add a small changelog entry.

Done. Please resolve if OK. BTW: what about the other direction, i.e. structuring? Anything to take care of there as well?

Reply:
Added it, works